### ISSUE TITLE
rancher cli does not honor app rollbacks

### ISSUE THREAD

**Issue by mshack70** [rancher/rancher#20750]
Rancher server 2.2.3
Rancher cli version 2.2.2

I am using a custom role user, with full project owner rights.  This is with a 'cluster' level catalog.

$ rancher app st helm-template
0.0.10
0.0.11
0.0.12
0.0.13
0.0.14
0.0.15
0.0.16
0.0.17
0.0.18
0.0.19
0.0.20
0.0.21
0.0.22
$ rancher app
ID             NAME      STATE     CATALOG            TEMPLATE        VERSION
p-nbp8p:mitt   mitt      active    mikehelm-cluster   helm-template   0.0.20
$ rancher app rollback mitt  -r
CURRENT   REVISION   CATALOG   TEMPLATE   VERSION   CREATED
$ rancher catalog refresh mikehelm-cluster

$ rancher app upgrade --values ../mitt-values.yaml mitt 0.0.21
$ rancher app
ID             NAME      STATE     CATALOG            TEMPLATE        VERSION
p-nbp8p:mitt   mitt      active    mikehelm-cluster   helm-template   0.0.21

$ rancher app rollback mitt  -r
CURRENT   REVISION   CATALOG   TEMPLATE   VERSION   CREATED

Also,  in UI, it does not show any rollback versions.  It does recognize all the upgrade versions.

**Comment by mshack70** [comment]
I'm testing also adding the app in the UI, and I see same behavior.  I can install/upgrade.  But if I try to rollback, I don't see any revisions to rollback to.

**Comment by ankur111** [comment]
https://github.com/rancher/rancher/issues/20204

**Comment by ankur111** [comment]
duplicate of the bug above.

**Comment by deniseschannon** [comment]
This is not a duplicate of #20204 as this is related to CLI.

**Comment by bmdepesa** [comment]
rancher/rancher:master-head bc3700f8

This was resolved by the fix for #20204

```
~/Dev/tools >> ./rancher app rollback docker-registry -r                                                    
CURRENT   REVISION            CATALOG   TEMPLATE          VERSION   CREATED
*         apprevision-k9xjg   library   docker-registry   1.6.1     07 Aug 2019 21:39:16 UTC
          apprevision-dffd5   library   docker-registry   1.6.1     07 Aug 2019 21:28:33 UTC
~/Dev/tools >> ./rancher app rollback docker-registry apprevision-dffd5              
~/Dev/tools >> ./rancher app rollback docker-registry -r                                                   
CURRENT   REVISION            CATALOG   TEMPLATE          VERSION   CREATED
*         apprevision-kkjnl   library   docker-registry   1.6.1     07 Aug 2019 21:40:05 UTC
          apprevision-k9xjg   library   docker-registry   1.6.1     07 Aug 2019 21:39:16 UTC
          apprevision-dffd5   library   docker-registry   1.6.1     07 Aug 2019 21:28:33 UTC
```
